packages/cli/src/options/beaconNodeOptions/chain.ts

```diff
@@ -123,11 +125,20 @@ Will double processing times. Use only for debugging purposes.",
     group: "chain",
   },
 
-  "chain.proposerBoostEnabled": {
+  "chain.proposerBoost": {
+    alias: "proposerBoostEnabled",
```

Comment:
```suggestion
    alias: ["chain.proposerBoostEnabled"],
```

doesn't need to be an array but slightly more consistent with existing aliases